src/main/scala/org/allenai/wikitables/TestWikiTablesCli.scala

```diff
@@ -57,22 +59,29 @@ class TestWikiTablesCli extends AbstractCli() {
     loader.done()
 
     // Read test data.
-    val testData = options.valuesOf(testDataOpt).asScala.flatMap(filename => {
-      WikiTablesUtil.loadDataset(filename, false, null, options.valueOf(maxDerivationsOpt))
-    })
+    val buffer = ListBuffer[(WikiTablesExample, RawEntityLinking)]()
+    for (filename <- options.valuesOf(testDataOpt).asScala) {
```

Comment:
Is there a reason you want a buffer instead of just doing a `.flatMap`?

You could instead make these lines like this:
```scala
val testData = options.valuesOf(testDataOpt).asScala.flatMap(filename => {
  val examples = WikiTablesUtil.loadDataset(
    filename, false, null, options.valueOf(maxDerivationsOpt))
  val linkings = entityLinker.loadDataset(filename, examples)
  val linkingsMap = linkings.map(x => (x.id, x)).toMap
  examples.map(x => (x, linkingsMap(x.id)))
})
```